test/commands/cat_tee_test.exs

```diff
@@ -1,44 +1,69 @@
 defmodule Cog.Test.Commands.CatTeeTest do
-  use Cog.AdapterCase, adapter: "test"
+  use Cog.CommandCase, command_tag: "cat-tee"
```

Comment:
The original test had them in the same module and I just copied that making the updates necessary to work in the new pattern. I can break them up though.